### PR TITLE
Find similar

### PR DESCRIPTION
**Pull request by grenlayk**

### CODE REVIEW

**Review by polinasand — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/MealServlet.java [hunk 1]
```diff
@@ -111,6 +119,18 @@ private void getMealById(HttpServletRequest request, HttpServletResponse respons
         return;
     }
 
+     private void returnUrlToSimilar(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // Fuction redirect to random page
+        // TODO(grenlayk): implement suggestions algorithm here for Product Alpha
+        Random rand = new Random(); 
+        int randomId = rand.nextInt(dishes.size()); 
+        String url = "/meal.html" + "?id=" + Integer.toString(randomId);
```
Comment (zajonc-gugle) [inline src/main/java/com/google/sps/servlets/MealServlet.java]: I believe you can use `StringBuilder` here, instead of concatenating string.

Comment (zajonc-gugle) [inline src/main/java/com/google/sps/servlets/MealServlet.java]: I'd suggest letting the frontend build a full url. It should be sufficient for the backend to just return the Id of a suggested meal.

**Review by zajonc-gugle — CHANGES_REQUESTED** [review]

File: src/main/webapp/meal-page-script.js [hunk 2]
```diff
@@ -55,7 +55,14 @@ function createElementByTag(text, tag) {
   return element;
 }
 
+function redirectToSimilar() {
+    fetch('/meal/similar').then(response => response.json()).then((id) => {
+        const url = '/meal.html?id=' + id.toString();
```
Comment (zajonc-gugle) [inline src/main/webapp/meal-page-script.js]: You can use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here.

File: src/main/java/com/google/sps/servlets/MealServlet.java [hunk 3]
```diff
@@ -111,6 +119,17 @@ private void getMealById(HttpServletRequest request, HttpServletResponse respons
         return;
     }
 
+     private void returnUrlToSimilar(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (zajonc-gugle) [inline src/main/java/com/google/sps/servlets/MealServlet.java]: You can rename the method now. It no longer returns a url.

**Review by zajonc-gugle — APPROVED** [review]

**Pull request merged**